.travis.yml

```diff
@@ -33,29 +34,31 @@ env:
 matrix:
   fast_finish: true
   include:
-    - php: '7.0'
+    - php: '7.1'
       env: TARGET=docs
-    - php: '7.0'
+    - php: '7.1'
       env: TARGET=lint
     - php: '5.6'
       env: COMPOSER_FLAGS="--prefer-lowest"
-    - php: '7.0'
+    - php: '7.1'
       env: SYMFONY=2.8.*
-    - php: '7.0'
+    - php: '7.1'
+      env: SYMFONY=3.0.*
```

Comment:
Wait, how come [7.0 is present in the dev-kit config for 1.x](https://github.com/sonata-project/dev-kit/blob/master/config/projects.yml#L183), [but missing from the generated (?) travis config](https://github.com/sonata-project/SonataDoctrinePhpcrAdminBundle/blob/1.x/.travis.yml)?